stdlib/2and3/logging/__init__.pyi

```diff
@@ -29,12 +29,13 @@ else:
     _levelNames = ...  # type: dict
 
 class Logger:
-    name = ...  # type: str
+    name = ...  # type: Text
```

Comment:
`Text` maps to `unicode`, on Python 2, but `name` can be a (byte)string, as well. I think `str` was a better annotation here, unless you actually want to create Loggers with a `unicode` name on Python 2.